docs/Encoder.md

```diff
@@ -1,57 +1,81 @@
 
-| Parameter | MIN            | MAX                | 
-|-------------------|-----------------|----------------------|
-| Supply voltage Vcc             |                 | 7 V          |
-| Input voltage (A, A̅, A, B̅, Z or Z̅)           | -11 V            | 14 V     |
-| Differential input voltage      | -14 V            | 14 V    |
-| Output voltage (OUT A, OUT B, OUT Z) | -0.5 V              | (Vcc + 0.5) V  |
+The AMDC REV D hardware supports two independent encoder interfaces. These must be incremental encoders outputting differential ABZ signals (i.e. the encoder outputs both the non-inverting and inverting signal, e.g. A and A̅). If the encoder only provides single-ended ABZ outputs, an adapter must be used between the encoder and the AMDC which converts the single-ended outputs to the differential signals required by the AMDC. For proper operation, the encoder ground should be connected to the AMDC ground via a pin on the DB9 connector. For powering the encoder, the user has two options: the AMDC provides 5 V power via the DB9 connector, or the user can use an external power supply.
 
-Exposure to absolute maximum ratings for extended period of time may affect device reliability. Therefore, it is recommended to operate under recommended operating conditions:
+### 2. DB9 Connector
 
-| Parameter | MIN            | NOM                | MAX|
-|-------------------|-----------------|----------------------|----------------------|
-| Supply voltage Vcc             | 4.5 V | 5 V          | 5.5 V |
-| High-level input voltage (logic HIGH) | 2 V |      | Vcc     |
-| Low-level input voltage (logic LOW)   | 0 V |      | 0.8 V    |
-| Common-mode input voltage | -7 V      |      | 7 V  |
+Each D-sub connecter has 9 pins – 6 for differential ABZ signals, 2 for ground signals and 1 for 5V supply to encoder. A jumper should be installed if the encoder is powered by 5 V from AMDC. There are two jumpers numbered on the silkscreen as JP2 and JP3 (each for one encoder input). 
 
-When chip is operated at nominal conditions (Vcc = 5 V, room temperature 25<sup>0</sup>), the output signals of the chip are such that the high-level output voltage is 3.8 V or more and low-level output voltage is around 0.2 V. Typical current the chip consumes is about 10 mA. This translates to about 50 mW of power consumption with 5 V supply. The rise and fall times of the output signal are both approximately equal to 4 ns. Assuming the system as first order, the bandwidth of the chip would approximately be equal to 0.35/4 ns = 87.5 MHz.
+A table of the pin mappings for each D-Sub connector is shown below:
 
-Before connecting encoder to AMDC, it should be checked whether encoder output signals satisfy recommended operating conditions of the chip. More detailed information on the operating conditions can be found in the [datasheet](http://www.ti.com/lit/ds/symlink/am26c32.pdf).
+| Pin number | Signal name |

```

Comment:
Should we be providing information about which picozed pins these are connected to in a manner that would help firmware developers?

I don't mean the physical pin numbers, as much as I mean the net name that Vivado uses when you set up your user constraints file to map Verilog code to the physical hardware. When manipulating the low level Verilog, it would be helpful to be able to quickly look up the encoder interface doc and be able to see a table of which nets the Verilog module should be mapped to.

Thoughts on this?

Reply:
Looks like it is addressed by @npetersen2 in 58e1b73